<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <widget class="QLineEdit" name="intervalEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
  </widget>
  <widget class="QLabel" name="tTimeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tot. Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>File Path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="intervalLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>T Interval</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tTimeEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>sec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lampStatus">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="setupButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="msLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>1036</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
